### ISSUE TITLE
RFC: Social Media Share Buttons for Posts

### ISSUE THREAD

**Issue by dwmkerr** [mismith0227/hugo_theme_pickles#153]
Hi @mismith0227 - love the theme, I use it on my dwmkerr.com website :smile:

I was wondering if you think this could be valuable - having social media share buttons on posts or pages themselves, so people can share a direct link to the page.

We could use the buttons a bit like here:

![image](https://user-images.githubusercontent.com/1926984/82400400-cbe1ee80-9a89-11ea-8e66-2d35d66c8e59.png)

But at the top and bottom of each post (optional, defaulting to off).

Would love to know if you think this would be useful, if so I can try and do it myself or if you have people who are keen to contribute we could mark as 'good first issue'? What do you think?

Thanks again for creating such a great theme.

**Comment by ckoehler** [comment]
Agreed. I've added buttons from https://sharingbuttons.io/ into my single template for now, but that'd be nice.
